nautobot/core/forms/fields.py

```diff
@@ -79,20 +78,6 @@ def __init__(self, from_form, *args, **kwargs):
                 "in double quotes."
             )
 
-    def to_python(self, value):
-        if value is None:
-            return None
-        reader = csv.reader(StringIO(value.strip()))
-        return forms.parse_csv(reader)
-
-    def validate(self, value):
-        if value is None:
-            return None
-        headers, _records = value
-        forms.validate_csv(headers, self.fields, self.required_fields)
```

Comment:
May be able to also delete the `parse_csv` and `validate_csv` helper functions as I believe they're now unused. A quick search of the github org doesn't show any apps using those functions directly.